pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -1914,6 +1914,22 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 			Expect(causes).To(BeEmpty())
 		})
 
+		It("should allow cdrom without a volume", func() {
+			vmi := api.NewMinimalVMI("testvmi")
+
+			vmi.Spec.Domain.Devices.Disks = append(vmi.Spec.Domain.Devices.Disks, v1.Disk{
+				Name: "testdisk",
+				DiskDevice: v1.DiskDevice{
+					CDRom: &v1.CDRomTarget{
+						Bus: v1.DiskBusSCSI,
+					},
+				},
+			})
+
```

Comment:
Can you please use the libvmi functions?

Reply:
sure